content/en/continuous_integration/troubleshooting.md

```diff
@@ -3,25 +3,37 @@ title: Troubleshooting CI Visibility
 kind: documentation
 ---
 
-### I instrumented my CI provider but I don't see any data in Datadog
-
-1. Make sure that at least one pipeline has finished executing. Pipeline execution information is only sent after it has finished.
-2. Check if you see any results in the [Pipeline Executions][1] section. In the [Pipelines][2] section, only pipelines that contain git information and that were executed in the default branch will be shown.
-3. If instrumenting Jenkins, make sure the Datadog Agent is running in the master host and available at its default location (`localhost:8126`).
-4. If you still don't see any results, [contact Support][3] and we'll help you troubleshoot.
+### I instrumented my Jenkins instance but I don't see any data in Datadog
```

Comment:
There is a troubleshooting section in Jenkins instructions. Would that section better for this?

If not, this is Jenkins specific and the other ones are more generic. It could be misleading because if the user doesn't read carefully, it might think that this applies to everything. Maybe different sections such as: `General`,  `Pipelines - Jenkins`, or something similar.

Reply:
This particular "issue" is not present on the Jenkins section. However, I would try to consolidate all troubleshooting here going forward, and if it gets too messy, we can add different sections.

Reply:
In that case, maybe we should consider moving troubleshooting from Jenkins to here.